mxenabled/connect-widget
# Don't show back button on search when heavy disclosure is on

Issue: [https://mxcom.atlassian.net/browse/CT-1079](https://mxcom.atlassian.net/browse/CT-1079)

### Testing instructions
- Turn on display_disclosure_in_connect flag and ensure search doesn't show a back button when you click continue button from the disclosure
- Run both Integration and E2E tests
